catkit/hardware/standa/stages.py

```diff
@@ -91,11 +98,19 @@ def home(self):
         hmst.uHomeDelta = int(self.u_homeOffset)

```

Comment:
@aidancgray This looked super suspicious so I tested it to make sure they weren't doing anything 🤯  and it doesn't work as suspected. I've added ``set_home_settings``.